src/Autosuggest.js

```diff
@@ -176,13 +179,14 @@ class Autosuggest extends Component {
     const {
       suggestions, renderSuggestion, inputProps, shouldRenderSuggestions,
       onSuggestionSelected, multiSection, renderSectionTitle, id,
-      getSectionSuggestions, focusInputOnSuggestionClick, theme, isFocused,
+      getSectionSuggestions, focusInputOnSuggestionClick, theme,isFocused,
```

Comment:
Erm. Feels like ESLint should have a rule for that but I haven't been able to find it.